### PR TITLE
Release aws-operator 3.1.0

### PR DESCRIPTION
**Pull request by corest**

### CODE REVIEW

**Comment by kopiczko** [comment]
AFAIK we didn't successfully run conformance tests on 1.10 cc @puja108

**Comment by xh3b4sd** [comment]
This might be related: https://github.com/giantswarm/giantswarm/issues/2999. At least on KVM this blocks releasing now.

**Comment by calvix** [comment]
done my pr is merged, this can proceed

**Comment by rossf7** [comment]
I still need to merge the Container Linux / Docker update. Also @r7vme is still working on the m5.large master change. So please can we hold off until they are in.

**Comment by rossf7** [comment]
OK I'm done too.

**Comment by r7vme** [comment]
Yes, please wait for my changes.

**Comment by r7vme** [comment]
My changes merged into master. Please go ahead.

File: service/controller/v10/version_bundle.go [hunk 1]
```diff
@@ -145,6 +145,6 @@ func VersionBundle() versionbundle.Bundle {
 		Name:         "aws-operator",
 		Time:         time.Date(2018, time.March, 22, 15, 47, 0, 0, time.UTC),
```
Comment (r7vme) [inline service/controller/v10/version_bundle.go]: Please change date of release.

We already voted about that and most of the people agreed to use date of release.

https://gigantic.slack.com/archives/C04TGHDEF/p1525071756000249

**Pull request merged**